src/python/WMComponent/AnalyticsDataCollector/DataCollectAPI.py

```diff
@@ -76,13 +69,13 @@ def getJobSummaryByWorkflowAndSite(self):
             for x in results.get('rows', []):
                 data.setdefault(x['key'][0], {})
                 data[x['key'][0]].setdefault(x['key'][2], {})
-                #data[x['key'][0]][x['key'][1]].setdefault(x['key'][2], {})
+                # data[x['key'][0]][x['key'][1]].setdefault(x['key'][2], {})
```

Comment:
Not that you should change this this time, but in general leaving commented out code sitting around with no indication of what it could be useful for is just clutter